mender/values.yaml

```diff
@@ -66,18 +66,19 @@ ingress:
   # alb.ingress.kubernetes.io/healthcheck-path: /ui/
   # cert-manager.io/issuer: "letsencrypt"
   path: /
-  extraPaths: []
+  extraPaths:
```

Comment:
Please add formatting to a different commit (or remove it entirely ;) )

Reply:
Done.